scripts/update-doc-yml.py

```diff
@@ -0,0 +1,38 @@
+"""
+Author: Jack He (jackhe@microsoft.com)
+
+This utility script is useful for creating a toc.yml file specifically for use with DocFx.
+
+DocFx is a dotnet tool for generating documentation from markdown files.
+It requires a toc.yml file to link markdowns to generate the static site.
+"""
+
```

Comment:
IMO, we should always use the already established scripting language, which is powershell.

Reply:
Sounds good. I can update.